### DISCUSSION TITLE
Geo Locations into Whatsapp

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1640] [Q&A]
After indexing an Itunes backup of an Iphone in version 4.1, in search I found the message below.
The Geo Location line called my attention.
What I wanted to know is what this location refers to.
As far as I know, whatsapp does not record the location of the origin of the message.
What exactly is this geolocation?

![gps](https://user-images.githubusercontent.com/381594/232347837-72a77ac6-7b0e-4187-8376-6347bc0d453b.png)

Tks
Paulo

**Comment by wladimirleite** [comment]
You can send your location through WhatsApp.

**Reply by paulobreim** [reply]
ahhh This information is from when someone sends her location to the other person!

Great. 
Thanks.
